templates/enterprise-license-job.yaml

```diff
@@ -92,7 +92,7 @@ spec:
                 chmod +x ./apply-license.sh
 
                 # Time out after 20 minutes. Use || to support new timeout versions that don't accept -t
-                timeout -t 1200 ./apply-license.sh 2> /dev/null || timeout 1200 ./apply-license.sh 2> /dev/null
+                timeout 1200 ./apply-license.sh || timeout -t 1200 ./apply-license.sh
```

Comment:
We did this so you don't see timeout usage errors unnecessarily. Didn't realize it was swallowing the error output from `consul license put` though.

I guess the only fix to ensure the timeout usage errors are silenced would be to capture the output from `consul license put` and redirect it to /stdout?

Reply:
Ah, that's true! Let me try and redirect the output.

Reply:
Output redirection is much better! I've switched to that.